modules/tools/prediction/mlp_train/scripts/records_to_data_for_learning.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+###############################################################################
+# Copyright 2018 The Apollo Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###############################################################################
+
+SRC_DIR=$1
+TARGET_DIR=$2
+
+set -e
+
+source /apollo/scripts/apollo_base.sh
+source /apollo/cyber/setup.bash
+
+sudo mkdir -p ${TARGET_DIR}
+
+if [ -z "$3" ]; then
+    MAP_DIR="sunnyvale_with_two_offices"
+else
+	MAP_DIR=$3
+fi
```

Comment:
Align the indentation with above line. Also, I'm not familiar with bash script, is there a ";" after the if statement correct?